server/src/main/java/org/candlepin/pinsetter/core/PinsetterKernel.java

```diff
@@ -67,6 +67,7 @@
 
     public static final String CRON_GROUP = "cron group";
     public static final String SINGLE_JOB_GROUP = "async group";
+    public static final String[] DELETED_JOBS = new String[] { "StatisticHistoryTask" };
```

Comment:
Is there someway we can introspect what the DELETED_JOBS should be instead of hard coding it?
Or possibly making it config?